downstreamadapter/dispatchermanager/event_dispatcher_manager.go

```diff
@@ -106,6 +111,10 @@ type EventDispatcherManager struct {
 
 	// sink is used to send all the events to the downstream.
 	sink sink.Sink
+	// redo related
+	redoSink     *redo.Sink
+	redoGlobalTs atomic.Uint64
+	redoMeta     *redo.RedoMeta
```

Comment:
redoMeta seems be a module of redo table trigger event dispatcher, not a part of event dispatcher manager.

Reply:
It looks simpler to add it in the dispatcher manager. and it only initiates when there exists a table trigger event dispatcher. The dispatcher does not need to know the existence of redo meta.